docs/root/intro/arch_overview/advanced/attributes.rst

```diff
@@ -0,0 +1,191 @@
+.. _arch_overview_attributes:
+
+Attributes
+==========
+
+Attributes refer to contextual properties provided by Envoy during request and
+connection processing. They are named by a dot-separated path (e.g.
+`request.path`), have a fixed type (e.g. `string` or `int`), and may be
+absent or present depending on the context. Attributes are exposed to CEL
+runtime in :ref:`RBAC filter <arch_overview_rbac>`, as well as Wasm extensions
+via `get_property` ABI method.
+
+Attribute value types are limited to:
+
+* `string` for UTF-8 strings
+* `bytes` for byte buffers
+* `int` for 64-bit signed integers
+* `uint` for 64-bit unsigned integers
+* `bool` for booleans
+* `list` for lists of values
+* `map` for associative arrays with string, integer, or boolean keys
```

Comment:
Do we ever use integer or boolean keys? The main reason I ask is that with ext_proc, this is a Struct, which is a JSON dictionary.

Reply:
No, we don't. I think it's fine to say it's always string keys.